src/main/java/seedu/address/logic/commands/FindTagsCommand.java

```diff
@@ -0,0 +1,38 @@
+package seedu.address.logic.commands;
+
+import seedu.address.model.person.TagsContainsKeywordsPredicate;
+
+/**
+ * Finds and lists all persons in address book whose tags contains any of the argument keywords.
+ * Keyword matching is case sensitive.
+ */
+public class FindTagsCommand extends Command {
+
+    public static final String COMMAND_WORD = "findtags";
+    public static final String COMMAND_ALIAS = "ft";
+
+    public static final String MESSAGE_USAGE = COMMAND_WORD + ": Finds all persons whose tags contain any of "
+            + "the specified keywords (case-sensitive) and displays them as a list with index numbers.\n"
```

Comment:
I think it would be better for the searching of tags to be case-insensitive, users might not remember which letters of the tags they capitalized especially when they have a lot of tagged contacts or if they created the tag a long time ago

Reply:
Apologies. It is already case insensitive, I must have missed out that part of the documentation.